asllib/doc/AssignableExpressions.tex

```diff
@@ -1633,6 +1710,48 @@ \subsection{Semantics}
 }
 \end{mathpar}
 
+\SemanticsRuleDef{LESetCollectionFields}
+
+\ProseParagraph
+\AllApply
+\begin{itemize}
+  \item $\vle$ is an expression for assigning each of the fields in $\fields$
+    of the collection global storage element $\vbase$ with the corresponding
+    slices given in $\vslices$ from the bitvector value $\vv$ (the rule
+    \TypingRuleRef{LESetFields} ensures that the length of $\fields$ and
+    $\vslices$ is the same);
+  \item view $\env$ as an environment where $\tenv$ is the static environment
+    and $\denv$ is the dynamic environment;
+  \item $\vbase$ is bound in the global dynamic environment ($G^\denv.\storage$);
+  \item $\vvone$ is the value of $\vbase$ in the global component of $\env$ ;
+  \item view $\vm$ as $(\vv, \vg)$;
+  \item view $\vrmrecord$ as $(\vvone, \{\})$;
```

Comment:
Is `\vm` defined? It looks like a free variable
And then should `\vmrecord` use `\Prosedefine` rather than "view"?